src/common/components/repositories-list/index.js

```diff
@@ -62,57 +64,51 @@ class RepositoriesList extends Component {
     }
   }
 
-  onPageLinkClick(pageNumber) {
-    this.props.dispatch(fetchUserRepositories(pageNumber));
-  }
-
   render() {
-    const isLoading = this.props.repositories.isFetching;
-    const pageLinks = this.props.repositories.pageLinks;
+    const isLoading = this.props.snaps.isFetching;
 
     return (
       <div>
-        { this.props.repositories.success && pageLinks &&
-          <div className={ styles['page-links-container'] }>
-            <PageLinks { ...pageLinks } onClick={ this.onPageLinkClick.bind(this) } />
-          </div>
-        }
         { isLoading &&
           <div className={ spinnerStyles }>
             <Spinner />
           </div>
         }
-        { this.props.repositories.success &&
-          this.props.repositories.repos.map(this.renderRepository.bind(this))
-        }
-        { this.props.repositories.success && pageLinks &&
-          <div className={ styles['page-links-container'] }>
-            <PageLinks { ...pageLinks } onClick={ this.onPageLinkClick.bind(this) } />
-          </div>
-        }
+        <Table>
+          <THead>
+            <TR>
+              <TH>Name</TH>
+              <TH>Configured</TH>
+              <TH>Registered for publishing</TH>
+              <TH>Latest build</TH>
+            </TR>
+          </THead>
+          <TBody>
+            { this.props.snaps.success &&
+              this.props.snaps.snaps.map(this.renderRow.bind(this))
+            }
+          </TBody>
+        </Table>
```

Comment:
This is my interpretation of a Vanilla table. I saw no point in giving components names that diverged from the existing table elements because tables are defined by their structure as well as their content.

Opinions welcome.